.eslintrc.js

```diff
@@ -2,31 +2,14 @@ module.exports = {
     env: {
         browser: true,
         es2021: true,
-        amd: true,
-        node: true,
-        'jest/globals': true,
     },
-    globals: {
-        Atomics: 'readonly',
-        SharedArrayBuffer: 'readonly',
-    },
-    extends: ['eslint:recommended', 'prettier'],
+    extends: ['airbnb-base'],
     parserOptions: {
-        ecmaVersion: 2018,
+        ecmaVersion: 12,
         sourceType: 'module',
     },
-    plugins: ['jest'],
     rules: {
-        'operator-linebreak': 0,
-        'no-bitwise': 0,
-        'import/no-unresolved': 0,
-        'implicit-arrow-linebreak': 0,
-        'no-else-return': 0,
-        'no-lonely-if': 0,
-        'space-unary-ops': 0,
-        'no-plusplus': 0,
-        'no-useless-constructor': 0,
-        'no-param-reassign': 0,
-        'import/extensions': 'off',
+        indent: ['error', 4],
+        'operator-linebreak': [2, 'after', { overrides: { '?': 'after', ':': 'after' } }],
```

Comment:
Need to add some rules and vue-config also for further plans. And also in this config, the parser should be changed to a typescript parser.

Reply:
why to typescript parser?

Reply:
Look, eslint doesn't understand your typescript code, so it needs a parser that translates your code to some abstract tree-like thing, and then validates your code.